ciao-controller/api/api.go

```diff
@@ -650,10 +669,18 @@ func Routes(config Config) *mux.Router {
 	route.Methods("POST")
 	route.HeadersRegexp("Content-Type", matchContent)
 
-	route = r.Handle("/{tenant:[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[8|9|aA|bB][a-f0-9]{3}-?[a-f0-9]{12}}/workloads", Handler{context, addWorkload})
```

Comment:
one or the other is going to get a merge conflict depending on who goes in first....